MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflEventWidget.ui

```diff
@@ -41,7 +138,45 @@
               </widget>
             </item>
             <item>
-              <spacer name="evenTimeSlicesSpacer">
+              <spacer name="customSpacer">
+                <property name="orientation">
+                  <enum>Qt::Horizontal</enum>
+                </property>
+                <property name="sizeHint" stdset="0">
+                  <size>
+                    <width>20</width>
+                    <height>20</height>
+                  </size>
+                </property>
+              </spacer>
+            </item>
+          </layout>
+        </widget>
+      </item>
+      <item>
+        <widget class="QGroupBox" name="logValueGroup">
+          <property name="title">
+            <string>Slicing by log value</string>
+          </property>
+          <layout class="QHBoxLayout" name="logValueLayout">
+            <item>
+              <widget class="QRadioButton" name="logValueButton">
+                <attribute name="buttonGroup">
+                  <string notr="true">slicingOptionsButtonGroup</string>
+                </attribute>
+              </widget>
+            </item>
+            <item>
+              <widget class="QComboBox" name="logValueComboBox">
+                <item>
+                  <property name="text">
+                    <string>T1</string>
+                  </property>
+                </item>
+              </widget>
+            </item>
+            <item>
```

Comment:
If slicing by log value is not implemented yet, can we remove the related widgets?